src/LongRange/LRHandlerBase.h

```diff
@@ -193,7 +193,7 @@ struct LRHandlerBase
       PosType k = kpts[ki];
       for (int iat = 0; iat < Zat.size(); iat++)
       {
-        grad1[iat] -= Zat[iat] * k * Fkg[ki] * (e2ikrA_r(iat, ki) * rhokB_i[ki] - e2ikrA_i(iat, ki) * rhokB_r[ki]);
```

Comment:
The change indicates a bug. Why you say it just makes the code more efficient?
If it is a bug, we need a test (hopefully unit test) to capture the bug and get pass after the fix.

Reply:
Sorry, a bad choice of words. By efficient I did not mean execution of the code but the workflow: fixing a bug such as this allows for more straightforward post-processing (and has to be done anyway). I'm currently working with @rcclay on an alternative fix using EwaldHandler3D, which apparently is better tested to start with. Will push it here shortly.

Reply:
@ye-luo LRHandlerBase has not been validated for forces--only EwaldHandler3D.  Ideally, this whole part of the code should be error trapped for users wanting to use forces. Since this is an experimental feature, this hasn't been done yet.

Reply:
@rcclay Could you comment a little more about the functionality of/distinction between LRHandlerBase and EwaldHandler3D?  What are the responsibilities of each?